packages/app/src/systems/Unlock/components/ResetDialog/ResetDialog.tsx

```diff
@@ -14,25 +24,38 @@ export function ResetDialog({
   onReset,
   children,
 }: ResetDialogProps) {
+  const [isSavedChecked, setSavedChecked] = useState(false);
+
   return (
     <Dialog>
       <Dialog.Trigger>{children}</Dialog.Trigger>
       <Dialog.Content css={styles.content}>
         <Dialog.Close />
         <Dialog.Heading>Forgot password</Dialog.Heading>
         <Dialog.Description as="div" css={styles.description}>
-          <Stack gap={'$2'}>
-            <Text>
-              If you have lost your password, the only way to recover your
-              wallet it is by using your seed phrase.{' '}
-            </Text>
-            <Text css={styles.warning}>
-              This action will remove all data stored on this device, including
-              your seed phrase, accounts, networks and other settings.
-            </Text>
+          <Stack gap={'$4'}>
             <Text>
+              If you lost your password, the only way to recover your wallet is
+              to delete all data and set up your wallet with your seed phrase.
               Make sure you have backed up your seed phrase before proceeding.
             </Text>
+            <Alert status="warning">
+              <Form.Control css={{ flexDirection: 'row' }}>
```

Comment:
```suggestion
              <Form.Control css={styles.form}>
```
css should be in an object